pl_examples/basic_examples/dali_image_classifier.py

```diff
@@ -93,15 +94,16 @@ def __init__(
             size=-1,
             reader_name=None,
             auto_reset=False,
-            fill_last_batch=True,
+            last_batch_policy=LastBatchPolicy.FILL,
             dynamic_shape=False,
             last_batch_padded=False,
     ):
-        super().__init__(pipelines, size, reader_name, auto_reset, fill_last_batch, dynamic_shape, last_batch_padded)
+        super().__init__(pipelines, size, reader_name, auto_reset, dynamic_shape,
+                         last_batch_policy=last_batch_policy, last_batch_padded=last_batch_padded)
 
     def __len__(self):
         batch_count = self._size // (self._num_gpus * self.batch_size)
-        last_batch = 1 if self._fill_last_batch else 0
+        last_batch = 0 if self._last_batch_policy == LastBatchPolicy.DROP else 1
```

Comment:
will this work also with older DALI versions, if not can we add a simple switch to something for `dali.__version__ > 0.28` and the old code for `<=0.27`

Reply:
No it will not. I will add some check based on Dali version like you said.

Reply:
I've pushed change for this. Tests passed except one which cancelled on its own.